### ISSUE TITLE
can someone provide install instructions for Ubuntu 16.0.4 LTS?

### ISSUE THREAD

**Issue by neogeo71** [BasioMeusPuga/Lector#20]
Would love to use this app, all efforts to install after installing dependencies have failed for me.

**Comment by BasioMeusPuga** [comment]
I find myself quite unfamiliar with how backporting happens for LTS releases. Have you been unable to find a ppa that has recent-ish versions of the dependencies we need?

**Comment by aethralis** [comment]
Perhaps a [snap package ](https://snapcraft.io/) would solve this issue?

**Comment by bjwest** [comment]
I agree with the snap package.  I'm using the latest Kubuntu (17.10), and it's latest PyQT5 is 5.7.  This gives me a 'TypeError: toFillPolygon(self, QTransform): not enough arguments' when I try to run it.  If I update to a newer PyQT5, what will that break on my system that currently uses the toFillPolygon with less arguments than the one in PyQT5 5.10?

**Comment by unhammer** [comment]
@bjwest I get that too, I think that warrants an issue of its own https://github.com/BasioMeusPuga/Lector/issues/51

**Comment by wordlessly** [comment]
Seems like a very nice ebook reader. Still, most people out there use Ubuntu 16.04 or some distribution based on it.

So all those people are locked out using this ebook reader :(

Why is it so hard to get this working..?

**Comment by CTAnthny** [comment]
Got this to run on Ubuntu Budgie 18.04 (GNOME) through manual build. Here's the commands if it helps others:
```shell
sudo apt update
sudo apt -y upgrade
python3 -V
sudo apt install -y python3-pip python3-dev python3-pyqt5 git
pip3 install pyqt5 lxml beautifulsoup4 xmltodict pymupdf setuptools
cd ~ && git clone https://github.com/BasioMeusPuga/Lector.git
cd Lector && python3 setup.py build
sudo python3 setup.py install
lector/__main__.py
```

**Comment by neogeo71** [comment]
these instructions worked. thank you.
